mmdet/integration/nncf/compression.py

```diff
@@ -61,6 +64,30 @@ def is_checkpoint_nncf(path):
     except FileNotFoundError:
         return False
 
+def get_nncf_config_from_meta(path):
+    logger = get_root_logger()
+    checkpoint = torch.load(path)
```

Comment:
Is there any way that does not require checkpoint reloading all the time?

Reply:
I did not think that this is a big issue.

Anyway, final loading of the model should be done _after_ the model is converted by NNCFNetwork, and if we should make this conversion (and how to make this conversion!) is detected by the configuration parameters. Now the parameters may be loaded from the meta of the checkpoint.

Reply:
Loading a huge file several times is not the thing that you want to do, usually.

Reply:
Agree. Could I write TODO here? I promise to fix this -- I will add this to my special TODO list.

Reply:
Yes.